apps/spruce/src/pages/waterfall/WaterfallGrid.tsx

```diff
@@ -104,18 +119,20 @@ export const WaterfallGrid: React.FC<WaterfallGridProps> = ({
           minOrder,
           revision,
           date: date ? fromZonedTime(date, timezone) : undefined,
+          ...serverFilters,
```

Comment:
idk why but when I visit mms with params `?buildVariants=ACS1&maxOrder=84102` the page just crashes for me 😭

I searched `"ACS1"` on mms then tried to go to the next page of results

Reply:
This should be addressed by https://github.com/evergreen-ci/evergreen/pull/8807 🤞

Reply:
Omg nvm DEVPROD-15622

Reply:
https://github.com/evergreen-ci/evergreen/pull/8808